src/app/kbv/controllers/load-question.js

```diff
@@ -7,6 +7,10 @@ const {
   },
 } = require("../../../lib/config");
 
+const questionKeyToPathMap = new Map([
+  ["rti-payslip-national-insurance", "enter-national-insurance-payslip"],
```

Comment:
I don't see any `enter-national-insurance-payslip` in the Imposter file, so I don't see how this will work just yet?

Reply:
The imposter file will still be using the question key, this is just to map question key to url path to get the correct route to go to next